cmd/flag/pass_range_flags.go

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2018 Infostellar, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flag
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Default time range used when end time is not specified.
+	durationInDays uint8 = 31
+	// Maximum value of duration in days.
+	maxDurationInDays uint8 = 31
+	// Time format used to parse "after" and "before" flags.
+	timeFormat = "2006-01-02 15:04:05"
```

Comment:
Should we make this `2006-01-02T15:04:05`? Otherwise users have to quote the string when specifying it as a flag on the command line since there is a space inside.

Reply:
This is used in an error message. We need a layout to format AOSAfter and AOSBefore when we want to show it in the message.